post/views.py

```diff
@@ -62,3 +64,35 @@ def update_post(request, id):
             return redirect("/")
     context = {"form": form, "post": post, "success_message": success_message}
     return render(request, "post/update_post.html", context)
+
+class AddCommentView(CreateView):
```

Comment:
I think we should only allow logged in people to add comments:)

Reply:
we do without saying .. because only log in ppl can see post ..etc, so I don't really need to add that, it's already required

Reply:
What I meant here, is that user can enter the url post/1/comment/ in the browser and add comment without being logged in or even registered.

Reply:
add @login_required